public/css/skeleton.css

```diff
@@ -321,6 +321,32 @@ a:hover {
   color: #0fa0ce;
 }
 
+/* Spacing
+–––––––––––––––––––––––––––––––––––––––––––––––––– */
+button,
+.button {
+  margin-bottom: 1rem;
+}
+
+input,
+textarea,
+select,
+fieldset {
+  margin-bottom: 1.5rem;
+}
+
+pre,
+blockquote,
+dl,
+figure,
+table,
+p,
+ul,
+ol,
+form {
+  margin-bottom: 2.5rem;
+}
+
```

Comment:
_Note:_ this change affects the specificity of the CSS rules **but** the UI didn't suffer from that change so I assumed it was safe to do.